shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.test.js

```diff
@@ -628,6 +628,51 @@ describe('serveCaseToIrsInteractor', () => {
     expect(applicationContext.getUtilities().uploadToS3).toHaveBeenCalled();
   });
 
+  it('should generate an order and upload it to s3 for orderToShowCause', async () => {
```

Comment:
Nit: This is quite a few expectation statements for one test, covering a lot of different AC. I would prefer to see them broken up into a few more tests that can have more specific names which will also help document the system's expected behavior.